debian/copyright

```diff
@@ -0,0 +1,41 @@
+Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: eos-data-distribution
+
+Files: *
+Copyright:
+ © 2016 Endless Mobile, Inc.
+License: LGPL-3+
+
+Files:
+ dbus-service.py
```

Comment:
To be clear: changing the copyrights in files should happen in a separate commit. I’m not going to do it in this commit, as it would confuse matters. The separate commit should update `debian/copyright`.